src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayCarRentCost = 40;
+  let discount = 0;
+
+  if (days >= 3 && days < 7) {
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md): Don't use literals in expressions (HARDCODE). Instead use constants explaining this values
![image](https://github.com/user-attachments/assets/f251c3af-36c4-4a8a-9dac-3af5b7d3ea48)